.kokoro/client-library-check.sh

```diff
@@ -122,9 +123,9 @@ fi
 
 pushd ${REPO}
 
-# TODO(#748): Replace the version of sdk-platform-java-config for all libraries. This logic will no longer
-#  be needed after the rest of the handwritten libraries are migrated to use this artifact.
-if [ "$REPO" == "java-storage" ]; then
+# If using an older version of java-storage, continue replacing java-shared-config version otherwise replace
+# the version of sdk-platform-java-config.
+if [ "${REPO_TAG}" == "v2.9.3" ] && [ "${REPO}" == "java-storage" ]; then
```

Comment:
This condition is needed until we can upgrade the flatten-plugin check to use at least v2.35.0 of java-storage which inherits sdk-platform-java-config instead of java-shared-config.